LoremFlickr support

mrstebo opened the pull request:
I was using `Faker::LoremPixel`, but you get a wider range of images using LoremFlickr.

You not only get grayscale images. You also get **pixelated** and **colorized** images.

b264 commented on lib/faker/lorem_flickr.rb:
```diff
@@ -0,0 +1,38 @@
+        build_url(size, 'p', search_terms, match_all)
+      end
+
+      def colorized_image(size = '300x300', color = 'red', search_terms = [], match_all = false)
```
I think search_terms needs a valid default value

mrstebo replied:
Yeah, I'll put something like `search_terms = ['faker']` :+1:

mrstebo commented:
I've refactored the `image` method and removed the `is_gray` parameter, and just added a separate `grayscale_image` method to make things a bit cleaner.

mrstebo commented:
Will do @vbrazo :+1:

vbrazo commented:
You should run `bundle exec rubocop -a` to fix the rubocop violations.

mrstebo commented:
@vbrazo should be all sorted now :raised_hands:
